docs/headless-form/index.md

```diff
@@ -0,0 +1,12 @@
+# Headless form integration
+
+Toucan Core form components are designed to work with [ember-headless-form](https://ember-headless-form.pages.dev/).
+
+Ember-headless-form includes integrations with
+
+- [native validation](https://developer.mozilla.org/en-US/docs/Web/HTML/Constraint_validation)
+- [changeset validation](https://github.com/poteto/ember-changeset)
+- custom validation
+- [yup](https://github.com/jquense/yup)
```

Comment:
Instead of linking to the libraries, we could also link to the docs for headless-form. We have a page for each of these bullet points, so that's a good fit. And they link to the pages you have now themselves, so you have even more context.